lib/Parser/XML/Xcal.rng

```diff
@@ -0,0 +1,1192 @@
+# RELAX NG Schema for iCalendar in XML
```

Comment:
Wondering if this should move outside of the lib/ tree... Perhaps good to have in a separate `schema` directory in the root? Are there any cons to that?